data/human/transport missions.txt

```diff
@@ -3582,3 +3582,743 @@ mission "Hauler VI cargo prototype 3"
+			choice
+				`	"Impressive."`
+				`	"It's OK."`
+			`	"It's just as I imagined it. Well, I have an exam to cram for. Enjoy the expo! I think it closes soon," she says as she heads for the exit.`
+			`	You wander around the exhibit in awe. It's a bit disconcerting as the occasional hologram walks through you. Inside houses you see families frantically gathering their belongings and running for the Navy ships. It strikes you that somewhere here is a young Katya leaving her doomed home. You walk down one of the lines of refugees, observing each person. An officer slips in the snow and a young man helps him up.`
+			`	As you move on you're startled to realize that you're face to face with the woman holding the baby, right where they were in the photo. She takes little steps toward the ship while bouncing the baby on her hip. A young woman with a camera and a respirator covering her face walks up to her, asks a question and begins to take photos. Before you can register what's going on the entire projection disappears in a disorienting flash, leaving you in a normal hanger bay. You are a little embarrassed to realize the exhibit is nearly empty as a woman walks up to you. "She loves that moment too. That's probably why she fought so hard to have that part left in, even though it's a security risk."`
+			`	"How rude of me," she says. "My name is Mariel, I organize these expositions and act as a spokesperson for Adelita. And you are?"`
+			choice
+				`	"Captain <last>."`
+			`	"Captain," she says, her eyes lighting up. "Well, as you may know this is the last day of the expo. The captain I hired to transport everything back to <destination> is not only rude but now late. I would much rather have it in the hands of someone who appreciates the work. I was going to pay him 60000, but will pay you 80000. What do you say?"`
+			choice
+				`	"I would love to."`
+				`	"Why not."`
+				`	"Sorry, I have more pressing matters at hand."`
+					goto oh
+			`	"Great," she says. "Go and bring your ship to this hanger. I will have the crew get everything ready to load."`
+				accept
+			label oh
+			`	"Oh, I'm surprised you would turn down so much money for such little work and a chance to meet the artists. But it's your call," she says. "The exhibit is closed. I'm going to have to ask you to leave."`
+				decline
+	on visit
+		dialog `You arrive on <planet>, but realize that your escort carrying the exposition has not arrived yet. Better depart and wait for your escorts to enter the system.`
+	on complete
+		payment 80000
+		log `Saw an impressive exposition on the Hope evaluation and ended up with a job taking the whole thing to Moonshake.`
+		conversation
+			`	You guide your ship down to the indicated hanger where there is a motley looking crew waiting to unload your ship. One of them hands you a note. It reads: "Meet me at the Common Grounds cafe in the spaceport if you're interested in doing a project with us. Make sure you're not followed. -Adelita Zero"`
```

Comment:
```suggestion
			`	You guide your ship down to the indicated hanger where there is a motley-looking crew waiting to unload your ship. One of them hands you a note. It reads: "Meet me at the Common Grounds cafe in the Moonshake spaceport if you're interested in doing a project with us. Make sure you're not followed. -Adelita Zero"`
```
You might want to consider adding some additional text at some point explaining why this semi-secretive organization is willing to trust a random captain.